File: Java/Array-2/bigDiff/BigDiff.java
package bigDiff;

public class BigDiff {
    //Retorne a diferença entre o maior e o menor número do Array dado ("nums").

    public int bigDiff(int[] nums) {
        int maior = nums[0], menor = maior;
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] > maior) maior = nums[i];
            if (nums[i] < menor) menor = nums[i];
        }
        return maior - menor;
    }
}

File: Python/List-2/big_diff.py
"""
Dada uma lista numérica de tamanho 1 ou mais "nums", retorne a diferença entre
o maior e menor valor da mesma.
"""


def big_diff(nums):
    return max(nums) - min(nums)


print(big_diff([1, 2, 20, 13, 7]))
